packages/emitter-framework/src/python/components/constant-declaration.tsx

```diff
@@ -0,0 +1,22 @@
+export interface ConstantDeclarationModel {
+  name: string;
+  value: string | number | undefined;
```

Comment:
Is leaving out a value allowed?

Reply:
Well, the concept of const in Python is a misnomer... you can totally overwrite a value. But the value could be "None". But you can't just have a naked variable declared without an assignment. For example you can't do the Python equivalent of:
```ts
let foo: string;
foo = "blah";
```